lsa-mis/lsa_ride_share
# Lra 577 lsa ride share survey faculty interface

This PR gives faculty (the current_user.uniqname) access to surveys that were created with that uniqname.
The index page for surveys:  http://localhost:3000/faculty_surveys/faculty_index (In the header, the user will see only this link.)
This link http://localhost:3000/faculty_surveys/survey/1 (1 is the survey_id) has list of questions with rich_text fields for answers. 
The FacultySurvey model has uniqname, term_is, and unit_id fields.
Every FacultySurvey record has_many :config_questions

The Survey module takes care of creating a survey page, saving the answers, creating a program from those answers, and validating the answers. 

To test:

- login as yourself 
- click on SuperAdmin in the header and turn on Faculty Survey preference for any Unit. 
- (you can create a Test Unit with test-unit-rideshare-admins ldap group - then you can test the application with um171712 user as the Test Unit admin)
- create a faculty survey with this uniqname um174887
- login as um174887
- I hope you'll see the Faculy Survey link in the header
- click on Questions link
- try to save the survey without any answers; only with the first two, etc

When the survey is saved you'll see the link to the created program and a button to Add Sites. The button is doing nothing for now. Editing a program by a faculty is a subject for the next pull request.
